web/httpheader.jspf

```diff
@@ -66,6 +67,13 @@ org.opensolaris.opengrok.web.PageConfig"
     <link rel="stylesheet" type="text/css" href="<%=styleDir%>/jquery.combo.css%>" />
 --%>
 <%
+    cfg.addScript(ctxPath + "/js/jquery-3.2.0.min.js", 0);
```

Comment:
Could be better solution as we dont need anything extra generic for our low number of js files. Will try.